doc/reference/javascript_reference.rst

```diff
@@ -2218,16 +2218,25 @@ in the action registry.
 
     .. code-block:: javascript
 
-        var ControlPanelMixin = require('web.ControlPanelMixin');
         var AbstractAction = require('web.AbstractAction');
 
-        var ClientAction = AbstractAction.extend(ControlPanelMixin, {
+        var ClientAction = AbstractAction.extend({
+            hasControlPanel: true,
+            loadControlPanel: true, // default: false
```

Comment:
Shouldn't this and the warning be placed in the "Using the control panel" section as it is more about how to use the Control Panel than implementing a new client action ?